opengrok-indexer/src/main/java/org/opengrok/indexer/web/SearchHelper.java

```diff
@@ -814,4 +814,61 @@ private void ensureSettingsHelper() {
             settingsHelper = new SettingsHelper(reader);
         }
     }
+
+    /**
+     * Search Helper Builder Class.
+     */
+    public static final class Builder {
+        private final SearchHelper searchHelper;
+
+        /**
+         * Search Helper Builder Class constructor.
+         * start parameter is defaulted to zero
```

Comment:
This is probably going to be rendered on single line in the javadoc. Also, it might be nicer if the parameter names were formatted distinctly.